frontend/src/pages/ActivityReport/Pages/Review/HtmlReviewItem.js

```diff
@@ -10,7 +9,7 @@ import { getEditorState } from '../../../../utils';
  * Reasoning for another component is to not overload `ReviewItem`
  */
 
```

Comment:
removed some code here to improve test coverage - we weren't using the path parameter anywhere we used HtmlReviewItem and it seemed a direct lift from the component we copied this one from